addons/map/README.md

```diff
@@ -8,6 +8,10 @@ Various tweaks to the in-game map, including:
 - Map illumination (optional)
 - Blue Force Tracker (optional)
 
+#### Variables
```

Comment:
It's a matter of readability to others editing the file, and for consistency. Doesn't matter if there is space or there isn't, but the 2 lines are different currently.

> If you don't want it there, I can remove it, though I don't think such a minor feature deserves a separate web page and a similar syntax is used in ACE3 already.

Everything _should_ be on the wiki. This variable API should be added to [Mission tools](http://ace3mod.com/wiki/missionmaker/mission-tools.html) page.